packages/multichain/src/scope/constants.ts

```diff
@@ -0,0 +1,47 @@
+import MetaMaskOpenRPCDocument from '@metamask/api-specs';
+import type { KnownCaipNamespace } from '@metamask/utils';
+
+// ScopeString for ecosystems that aren't chain specific
+export enum KnownWalletScopeString {
+  Eip155 = 'wallet:eip155',
+}
+
+// Known CAIP Namespaces excluding "wallet"
+export type NonWalletKnownCaipNamespace = Exclude<
+  KnownCaipNamespace,
+  KnownCaipNamespace.Wallet
+>;
+
+// Methods that do not belong to an ecosystem
+export const KnownWalletRpcMethods: string[] = [
+  'wallet_registerOnboarding',
+  'wallet_scanQRCode',
```

Comment:
Hmm. Technically it was introduced by an EIP, but it does seem useful outside of an Ethereum context. So maybe that's reason enough for it to belong here.

Reply:
Yeah that was our thinking